packages/x-date-pickers/src/internals/hooks/validation/useDateValidation.ts

```diff
@@ -25,14 +26,16 @@ export const validateDate: Validator<any, DateValidationProps<any, any>> = ({
 }): DateValidationError => {
   const now = adapter.utils.date()!;
   const date = adapter.utils.date(value);
-
-  const {
-    shouldDisableDate,
-    minDate = adapter.defaultDates.minDate,
-    maxDate = adapter.defaultDates.maxDate,
-    disableFuture,
-    disablePast,
-  } = props;
+  const minDate = parsePickerInputValueWithDefault(
```

Comment:
This value should already be normalised, because hooks are called after `use[...]PickerDefaultizedProps`